libraries/protocol/include/graphene/protocol/asset_ops.hpp

```diff
@@ -523,6 +527,7 @@ FC_REFLECT( graphene::protocol::asset_claim_pool_operation::fee_parameters_type,
 FC_REFLECT( graphene::protocol::asset_options,
             (max_supply)
             (market_fee_percent)
+            (taker_fee_percent)
```

Comment:
// BSIP81 maker/taker; percent of market fee

Reply:
I think this commentary would be redundant of the commentary where the variables are declared